client/extensions/zoninator/state/data-layer/zones/index.js

```diff
@@ -108,17 +108,18 @@ export const saveZone = ( { dispatch }, action ) => {
 	);
 };
 
-export const announceZoneSaved = ( dispatch, { form, siteId }, zone ) => {
+const announceZoneSaved = ( dispatch, { form, siteId }, zone ) => {
 	dispatch( stopSubmit( form ) );
 	dispatch( updateZone( siteId, zone.id, zone ) );
 	dispatch( successNotice( translate( 'Zone saved!' ), { id: saveZoneNotice } ) );
 };
 
 export const handleZoneCreated = ( { dispatch, getState }, action, response ) => {
-	const { siteId } = action;
+	const siteSlug = getSiteSlug( getState(), action.siteId );
```

Comment:
So we're generally trying to avoid `getState()` in actions when easily feasible -- e.g. where the action consumer can pass in the information we'd be otherwise gathering from state. (I've noticed it's used to determine the newly added zone as well.)

Here's the "canonical" source, a [SO question](https://stackoverflow.com/questions/35667249/accessing-redux-state-in-an-action-creator/35674575) with an answer by Redux creator Dan Abramov,